Automattic/wp-calypso
# 100-year-plan: Style thank you step and handle on appointment schedule event

Related to #

## Proposed Changes

- Matched "Thank You" step design

Figma: Qn8lSkQCQHwd2xZPaMTYjI-fi-5910_3400

## Testing Instructions

* Go to /setup/hundred-year-plan/diy-or-difm?flags=100year%2Fvip
* Select a time slot
* Check if there is "Thank you" step at the end of the flow
* Check the if "View guide" button leads to support doc page

NOTE: The button style has not been updated to match the design; I wanted to be consistent and use it from the `@wordpress/components` package. But I'll check with the designer and probably match it in the next PR.


<img width="1840" alt="Screenshot 2024-10-09 at 23 11 12" src="https://github.com/user-attachments/assets/fd0a092b-f417-40fa-8aee-b1211a8fa56c">


## Pre-merge Checklist
